ui/app/adapters/pki/issuer.js

```diff
@@ -39,8 +39,13 @@ export default class PkiIssuerAdapter extends ApplicationAdapter {
       const issuerRecord = await this.queryRecord(store, type, { id, backend: query.backend });
       const { data } = issuerRecord;
       const isRoot = await verifyCertificates(data.certificate, data.certificate);
-      const parsedCertificate = await parseCertificate(data.certificate);
-      return { ...keyInfo, ...data, isRoot, parsedCertificate };
+      const parsedCertificate = parseCertificate(data.certificate);
+      return {
+        ...keyInfo,
+        ...data,
+        isRoot,
+        parsedCertificate: { common_name: parsedCertificate.common_name },
```

Comment:
since commonName is also an attribute on the issuer (it's the only one that gets pulled out of the `parsedCertificate` object) I think this could also be 
```suggestion
        commonName: parsedCertificate.common_name ,
```

but no need to change, mostly just an FYI 😄